src/components/event-delegation.js

```diff
@@ -10,8 +10,12 @@ var listenersAttachedKey = '$MED' + runtimeId;
 function getEventFromEl(el, eventName) {
     var virtualProps = getMarkoPropsFromEl(el);
     var eventInfo = virtualProps[eventName];
+
     if (typeof eventInfo === 'string') {
         eventInfo = eventInfo.split(' ');
+        if (eventInfo[2]) {
+            eventInfo[2] = eventInfo[2] === 'true';
```

Comment:
We could save some extra bytes if we serialize this to something other than the string 'true'.
Probably doesn't need to be addressed in this PR though.